msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -60,6 +82,19 @@ To check whether the RSC permissions are being honored by the API request payloa
     4. Select **Get link to team**. 
     5. Copy and save the **groupId** value from the string.
 
+## Test added RSC permissions to a chat using the Postman app
+
+To check whether the RSC permissions are being honored by the API request payload, you need to copy the [RSC JSON test code for chats](test-chat-rsc-json-file.md) into your local environment and update the following values:
+
+* `azureADAppId`: Your app's Azure AD app ID.
+* `azureADAppSecret`: Your Azure AD app password.
+* `token_scope`: The scope is required to get a token. set the value to https://graph.microsoft.com/.default.
+* `chatId`: You can get the chat thread id from the Teams web client as follows:
+
+    1. In the Teams web client, select **Chat** from the far left navigation bar.
+    2. Select the chat where the app is installed from the dropdown menu.
+    3. Copy the web URL and save the chat thread id from the string.
+
```

Comment:
You could add a screenshot over here of a web address url with the url for a chat, and then highlight the substring that maps to the chat thread id.